src/components/molecules/AboutMethod/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC } from "react";
+import { Link } from "react-router-dom";
+
+const AboutMethod: FC = () => {
```

Comment:
How about we have a convention that we keep those views/pages in `pages/` directory? We could also agree on a naming convention, e.g.:
`AboutMethodView` or `AboutMethodPage` and align all page-like components also `ProblemIndexPage` or accordingly `ProblemIndexView`.
Btw I would rename it to `ProblemsView` or `ProblemsPage`.

Reply:
Good point, I've renamed them to "...`Page`" and moved to `pages/` directory :)